src/util/Injected.js

```diff
@@ -271,6 +271,7 @@ exports.LoadUtils = () => {
             ...ephemeralFields,
             ...locationOptions,
             ...attOptions,
```

Comment:
And why we need first ...attOptions ?

Reply:
because the first one includes the original mirrored content and the other has the property that will override the type. Overriding the type is how you fix this bug.

Reply:
I'm not sure, but it looks like necessary. I've tried without '...attOptions' and it's fine. But I don't use stickers. It seems that toJson() or serialize() converts object's fields (e.g. __x_type=> type) and this will not override